README.md

```diff
@@ -86,17 +86,17 @@ httpServer.listen(3000);
 
 ```javascript
 angular.module('rethinkDBWorkshop.services', [])
-  .factory('messageFactory', refactoryFactory, function (refactoryFactory) {
+  .factory('messageFactory', ['refactoryFactory', function (refactoryFactory) {
       return refactoryFactory({
```

Comment:
Is this what you meant?  I haven't seen the .factory method take three arguments before.